src/main/kotlin/blackjack/domain/player/PlayerBetAmount.kt

```diff
@@ -0,0 +1,27 @@
+package blackjack.domain.player
+
+data class PlayerBetAmount(val initValue: Long = INIT_BET_AMOUNT) {
+    private var revenue: Long = INIT_REVENUE
+
+    fun getRevenue(): Long {
+        return revenue
+    }
```

Comment:
```suggestion
    var revenue: Long = INIT_REVENUE
        private set
```

set에 대에서만 private을 하면 될 것 같아요~

Reply:
훨씬 간단하게 리팩토링이 가능하군요
감사합니다🙌